src/referrals-queue/referrals-queue.component.tsx

```diff
@@ -53,21 +54,32 @@ export default function ReferralsQueue(props: ReferralsQueueProps) {
           <div className={styles.inputContainer}>
             <div className={styles.dateInputContainer}>
               <label htmlFor="from-date">
-                <Trans i18nKey="from">Filter by date</Trans>
+                <Trans i18nKey="from">From</Trans>
               </label>
-              <DateRangePicker
-                startDate={fromDate}
-                startDateId="from_date"
-                endDate={toDate}
-                endDateId="to_date"
-                onDatesChange={({ startDate, endDate }) => {
-                  setFromDate(startDate);
-                  setToDate(endDate);
-                }}
-                focusedInput={focusedDateInput}
-                onFocusChange={i => setFocusedDateInput(i)}
+              <SingleDatePicker
+                id="from-date"
+                date={fromDate}
+                onDateChange={date => setFromDate(date)}
+                focused={fromDateFocused}
+                onFocusChange={({ focused }) => setFromDateFocused(focused)}
                 isOutsideRange={(date: Moment) => date.isAfter(today)}
                 displayFormat="YYYY MMM DD"
```

Comment:
I didn't notice this last time until afterwards, but I've got a slight preference for DD MMM YYYY, that's generally our standard.

Reply:
Good catch, thanks!